hash/spectrum_wafadrive.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+
+<!--
+	Wafadrive 'cartridges'
+	These are infinite tape loop based devices that act like discs
+-->
+<softwarelist name="spectrum_wafadrive" description="Sinclair ZX Wafadrive tapes/cartridges">
+
+	<software name="spctrlwr" supported="no"> 
+		<description>Spectral Writer</description>
+		<year>1984</year>
+		<publisher>	Softek International Ltd</publisher>
```

Comment:
Are there some unwantes spaces on the publisher tag?

Reply:
stray tab, fixed